tests/trait-tests-trait.php

```diff
@@ -0,0 +1,48 @@
+<?php
```

Comment:
This file's name looks wrong to me (having `trait` repeated twice). For instance, our `Parsely` class lives in `class-parsely.php` but the class is not named `Parsely_Class`. Maybe we could rethink how we name the class, and then rename the file accordingly?

Reply:
I've named it like that because I've tried to do what I've seen in general and bridge it with WordPress conventions. Maybe you would prefer `trait-tests.php` and rename the trait to `Trait_Tests`? `Tests` by itself is not semantic enough, so in that case we should go for another name altogether.

It's a matter of how we name things and I have no strong opinion on this. Regarding traits, this is what we have so far (from https://make.wordpress.org/core/2020/03/20/updating-the-coding-standards-for-modern-php/):

>  Trait and interface names should use capitalized words separated by underscores, same as class names. Any acronyms should be all upper case.
> - The word “Trait” is not required in a trait name, but won’t be forbidden either.
> - Trait file names should be based on the trait name with trait- prepended and the underscores in the trait name replaced with hyphens.

Reply:
This trait is not a general trait (for now). What it's doing is providing functions to change the visibility of methods inside a class. What about we rename it so something like `Tests_Visiblity` and then the file name would be `trait-tests-visibility.php`? 

Open to different names, but I don't know if the idea makes sense.

Reply:
I had opted for the name to be general (in order to avoid renames in the future), but this makes sense for now so I'll go with this.

Reply:
I went ahead and renamed this to `Tests_Reflection`, as a placeholder for all reflection methods for tests.